specification/containerservice/resource-manager/Microsoft.ContainerService/aks/preview/2025-01-02-preview/managedClusters.json

```
@@ -4766,6 +4766,11 @@
           "type": "string",
           "title": "The maximum number or percentage of nodes that can be simultaneously unavailable during upgrade.",
           "description": "This can either be set to an integer (e.g. '1') or a percentage (e.g. '5%'). If a percentage is specified, it is the percentage of the total agent pool size at the time of the upgrade. For percentages, fractional nodes are rounded up. If not specified, the default is 0. For more information, including best practices, see: https://learn.microsoft.com/en-us/azure/aks/upgrade-cluster"
+        }
```

Please fix failed CI check, the file is not a valid json file.
```suggestion
        },
```